src/components/ViewTrackerButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { useState, useEffect } from 'react';
+import styles from '../styles/LandingPage.module.less';
+import { useNavigate } from 'react-router-dom';
+import { InternalRoutes } from '../routes';
+
+interface Props {
+    content: string;
+}
+
+const ViewTrackerButton: React.FC<Props> = (props: Props) => {
+    const { content }: Props = props 
+    const navigate = useNavigate()
+    return (
+        <button className={styles.button} onClick={() => navigate(InternalRoutes.TRACKER.path)}>{content}</button>
+    );
```

Comment:
Can you use the antd button so that it gets the nice hover effect? Optionally you could manually do it in CSS but using antd is probably easier. 

https://ant.design/components/button/

Reply:
On it, also I think there is a separate task for getting the image behind the menu

Reply:
> Can you use the antd button so that it gets the nice hover effect? 
> 
> https://ant.design/components/button/

Looks like the button component with antd doesn't have a height value, only width, so I wouldn't be able to create it according to the wireframe

Reply:
Okay, can always override the CSS but your button is basically the same now (:

Reply:
to be honest i'd also prefer antd since it centers the text easier and has a clean animation, could you point me in the right direction on resizing it? I tried and it didn't work with css but maybe I was doing it wrong

Reply:
I think what you have right now is fine for now. We can come back to swapping the button out. Just put it in the backlog!